src/js/plugin-bridge/PluginSDK.js

```diff
@@ -96,8 +97,10 @@ const initialize = function (pluginsConfig) {
   replaceStoreReducers();
 
   // Allows plugins to do things before the application ever renders
-  let promises = Hooks.applyFilter('pluginsLoadedCheck', []);
-  Promise.all(promises).then(Hooks.notifyPluginsLoaded.bind(Hooks));
+  let promises = hooks.applyFilter('pluginsLoadedCheck', []);
+  Promise.all(promises).then(function () {
+    hooks.doAction('pluginsConfigured');
```

Comment:
Since we have the ability to remove actions, this now just calls an action instead of having a dedicated method